### PR TITLE
Feat: single-stake/no-rewards farm deployment scripts 

### PR DESCRIPTION
**Pull request by 0xp3th1um**

### CODE REVIEW

File: script/output/314311/phase-1b/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Script outputs for Mainnet for `phase-1b`.
```
Comment (amusingaxl) [inline script/output/314311/phase-1b/README.md]:
```suggestion
Script outputs for Tenderly Testnet for `phase-1b`.
```

File: script/input/314311/phase-1b/template-farming-deploy.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "nst": "address: the address for NST"
+
```
Comment (amusingaxl) [inline script/input/314311/phase-1b/template-farming-deploy.json]:
```suggestion  
```

File: script/input/1/phase-1b/template-farming-deploy.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "nst": "address: the address for NST"
+
```
Comment (amusingaxl) [inline script/input/1/phase-1b/template-farming-deploy.json]: Nit:
```suggestion
```

File: script/input/314311/phase-1b/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Script inputs for Mainnet for `phase-1b`.
```
Comment (amusingaxl) [inline script/input/314311/phase-1b/README.md]:
```suggestion
Script inputs for Tenderly Testnet for `phase-1b`.
```

File: script/phase-1b/01-FarmingDeploy.s.sol [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    ChainlogLike internal constant chainlog = ChainlogLike(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+
+    function run() external {
+        Reader reader = new Reader(ScriptTools.loadConfig());
+
+        address admin = chainlog.getAddress("MCD_PAUSE_PROXY");
+        address nst = reader.envOrReadAddress("FOUNDRY_NST", ".nst");
+
+        vm.startBroadcast();
+
```
Comment (amusingaxl) [inline script/phase-1b/01-FarmingDeploy.s.sol]: Dangling whitespace here:
```suggestion

```

**Review by amusingaxl — COMMENTED** [review]
A couple of nits, other than that, it looks good.

**Review by amusingaxl — APPROVED** [review]
LGTM

**Review by 0xdecr1pto — APPROVED** [review]

**Pull request merged**